src/hotspot/share/opto/callnode.hpp

```diff
@@ -656,7 +656,7 @@ class CallNode : public SafePointNode {
   virtual void copy_call_debug_info(PhaseIterGVN* phase, SafePointNode* sfpt) {}
 
 #ifndef PRODUCT
-  virtual void        dump_req(outputStream* st = tty) const;
+  virtual void        dump_req(outputStream *st = tty, DumpConfig* dc = nullptr) const;
```

Comment:
`*` should be left at the type:
```suggestion
  virtual void        dump_req(outputStream* st = tty, DumpConfig* dc = nullptr) const;
```

Reply:
done